src/renderer/components/AccountListSidebar/AccountItem.tsx

```diff
@@ -82,6 +98,40 @@ export default function AccountItem({
     },
   ])
 
+  let badgeContent
+
+  if (bgSyncDisabled) {
+    badgeContent = (
+      <div
+        className={styles.bgSyncDisabled}
+        aria-label='Background sync disabled'
+      >
+        ⏻
+      </div>
+    )
+  } else if (unreadCount > 0) {
+    badgeContent = (
+      <div
+        className={classNames(styles.freshMessageCounter, {
+          [styles.accountBadgeMuted]: muted,
+        })}
+      >
+        {unreadCount}
+      </div>
+    )
+  } else if (muted) {
+    badgeContent = (
+      <div
+        className={classNames(styles.accountMuted, {
+          [styles.accountBadgeMuted]: muted,
+        })}
+        aria-label='Account Notifications Muted'
+      >
+        <div className={styles.accountMutedIcon} />
+      </div>
+    )
+  }
```

Comment:
for muted accounts the switch from grey-badge to the mute icon is unexpected. also, the other way round, it looks as if mute is suddenly disabled.

it would be better to show the icon stable inside the avatar. this also looks more like a state and is also consisten with what we're doing on chats.

i suggest sth like the following, the icon could also be black with white drop shadow:

![Screenshot 2024-02-12 at 19 18 16](https://github.com/deltachat/deltachat-desktop/assets/9800740/d8a3779b-959a-4807-a675-08fdc8ddc359)